skeleton/Dockerfile

```diff
@@ -1,8 +1,8 @@
-ARG APP_INSIGHTS_AGENT_VERSION=2.6.3
+ARG APP_INSIGHTS_AGENT_VERSION=3.2.10
 
 # Application image
 
-FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.4
+FROM hmctspublic.azurecr.io/base/java:openjdk-17-distroless
```

Comment:
I don't think this is the right image

it should be using:
https://github.com/hmcts/draft-store/blob/master/Dockerfile#L3

you could probably add the multi-arch stuff as well the @TomKimber  did?
see https://github.com/hmcts/draft-store/pull/1006